### PR TITLE
Nrunner: fix example job

### PR DESCRIPTION
**Pull request by clebergnu**
The current example job that users the nrunner is currently broken, and can't be run.  This "fixes" it (considering the amount of limitation it still has).

Before this, one should get:

```
python3 ./examples/job/nrunner.py
JOB ID     : 0a6d504dbc164e1447c8dd89beccf216c2116b32
JOB LOG    : /home/cleber/avocado/job-results/job-2020-04-18T12.24-0a6d504/job.log
 (1/12) 1-selftests.unit.test_resolver.ReferenceResolution.test_no_args:  
Avocado crashed: ValueError: Unsupported kind of runnable: python-unittest
Traceback (most recent call last):

  File "/home/cleber/src/avocado/avocado/avocado/core/job.py", line 616, in run_tests
    execution_order)

  File "/home/cleber/src/avocado/avocado/avocado/plugins/runner_nrunner.py", line 57, in run_suite
    for status in task.run():

  File "/home/cleber/src/avocado/avocado/avocado/core/nrunner.py", line 759, in run
    runner_klass = self.runnable.pick_runner_class(self.known_runners)

  File "/home/cleber/src/avocado/avocado/avocado/core/nrunner.py", line 385, in pick_runner_class
    raise ValueError('Unsupported kind of runnable: %s' % self.kind)

ValueError: Unsupported kind of runnable: python-unittest

Please include the traceback info and command line used on your bug report
Report bugs visiting https://github.com/avocado-framework/avocado/issues/new
```

But after, it should result in:

```
./examples/jobs/nrunner.py 
JOB ID     : 75e0cb95532f62e8054fc75d462bd1add16502d4
JOB LOG    : /home/cleber/avocado/job-results/job-2020-04-18T12.24-75e0cb9/job.log
 (1/12) 1-selftests.unit.test_resolver.ReferenceResolution.test_no_args: PASS (0.01 s)
 (2/12) 2-selftests.unit.test_resolver.ReferenceResolution.test_no_result: PASS (0.01 s)
 (3/12) 3-selftests.unit.test_resolver.ReferenceResolution.test_no_resolutions: PASS (0.01 s)
 (4/12) 4-selftests.unit.test_resolver.AvocadoInstrumented.test_passtest: PASS (0.01 s)
 (5/12) 5-selftests.unit.test_resolver.AvocadoInstrumented.test_passtest_filter_found: PASS (0.01 s)
 (6/12) 6-selftests.unit.test_resolver.AvocadoInstrumented.test_passtest_filter_notfound: PASS (0.01 s)
 (7/12) 7-selftests.functional.test_argument_parsing.ArgumentParsingTest.test_unknown_command: PASS (0.68 s)
 (8/12) 8-selftests.functional.test_argument_parsing.ArgumentParsingTest.test_known_command_bad_choice: PASS (0.69 s)
 (9/12) 9-selftests.functional.test_argument_parsing.ArgumentParsingTest.test_known_command_bad_argument: PASS (0.68 s)
 (10/12) 10-selftests.functional.test_argument_parsing.ArgumentParsingErrorEarlyTest.test_whacky_option: PASS (0.66 s)
 (11/12) 11-/bin/true: PASS (0.01 s)
 (12/12) 12-/bin/false: FAIL (0.01 s)
RESULTS    : PASS 11 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```

### CODE REVIEW

File: avocado/plugins/runner_nrunner.py [hunk 1]
```diff
@@ -61,7 +61,10 @@ def run_suite(self, job, result, test_suite, variants, timeout=0,
                     break
 
             # test execution time is currently missing
-            test_state = {'status': statuses[-1]['status'].upper()}
+            # since 358e800e81 all runners all produce the result in a key called
+            # 'result', instead of 'status'.  But the Avocado result plugins rely
+            # on the current runner approach
+            test_state = {'status': statuses[-1]['result'].upper()}
```
Comment (beraldoleal) [inline avocado/plugins/runner_nrunner.py:64]: Hi @clebergnu, just one question: How difficult it is to fix that on the result plugin? IIUC, this ambiguity should be fixed on the upper layers too, right? Or shouldn't it?

File: avocado/plugins/runner_nrunner.py [hunk 2]
```diff
@@ -19,23 +19,20 @@
 from avocado.core import test
 from avocado.core.plugin_interfaces import Runner as RunnerInterface
 
-from avocado.core.nrunner import check_tasks_requirements
+from avocado.core import nrunner
```
Comment (beraldoleal) [inline avocado/plugins/runner_nrunner.py:22]: Currently, we are using only `check_tasks_requirements` and `RUNNERS_REGISTRY_PYTHON_CLASS`. Any reason to import the entire module here?

File: avocado/plugins/runner_nrunner.py (hunk 1 above)
Comment (clebergnu) [reply avocado/plugins/runner_nrunner.py:64]: The ambiguity is really caused by two different "standards". The current runner uses `status`, while the nrunner recently changed to use `result`. In theory we could change the current runner too, but I don't think that's the right approach at this point.

File: avocado/plugins/runner_nrunner.py (hunk 2 above)
Comment (clebergnu) [reply avocado/plugins/runner_nrunner.py:22]: Nothing really extraordinary... Best explanation is this being part of my WiP, which is going to use more symbols from the nrunner module.

**Pull request merged**